client/components/NewTestPlanReport/NewTestPlanReportModal.jsx

```diff
@@ -176,7 +176,11 @@ const NewTestPlanReportModal = ({
                                 key={`${item.gitSha}-${item.id}`}
                                 value={item.id}
                             >
-                                {item.gitSha}
+                                {calculateDateInterval(
+                                    new Date(item.updatedAt),
+                                    new Date()
+                                )}{' '}
+                                ({item.gitSha.substring(0, 5)})
```

Comment:
Assuming we keep the shas at all. Seems like we've decided to drop them.

Reply:
@s3ththompson are we keeping the SHAs?

Reply:
Sorry for not being clearer... Matt's feedback was to remove SHAs from the TestQueueRows, but keep them in the NewTestPlanReportModal:

- TestQueueRows:
    - `Nov 18, 2021 at 1:31pm`
- NewTestPlanReportModal:
    - `Nov 18, 2021 at 1:31pm (sha1234)` or
    - `Nov 18, 2021 at 1:31pm "Create tests for APG design pattern example: Main Landmark" (sha1234)`
   
Matt expressed that if we could add commit messages to the modal that would be even better, but I think we should do whatever is easier here since this is a temporary fix.